index.html

```diff
@@ -1,255 +1,418 @@
-                </li>
-                <li>
-                    <a href="#"><img src="images/linkedn.svg" alt="linkedn icons"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="images/medium.svg" alt="medium icons"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="images/github.svg" alt="github icons"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="images/finger.svg" alt="finger icons"></a>
-                </li>
+      </header>
+     
+
+
+      <!-- Header-section Ends here -->
+      <!-- Home-page section Starts here -->
+      <main>
+        <section class="home-page container">
+          <div class="home-title">
+            <h1>
+              I’m Samuel <br />
+              Glad to see you!
+            </h1>
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <a href="#" class="connect">LET'S CONNECT</a>
+
+            <ul class="social-icons">
+              <li>
+                <a href="twitter.com/" target="_blank">
+                  <img src="./images/twitter.svg" alt="twitter-icon" />
+                </a>
+              </li>
+
+              <li>
+                <a href="linkedin.com" target="_blank">
+                  <img src="./images/linkedIn.svg" alt="linkedin-icon"
+                /></a>
+              </li>
+
+              <li>
+                <a href="medium.com" target="_blank">
+                  <img src="./images/medium.svg" alt="medium-icon" />
+                </a>
+              </li>
+
+              <li>
+                <a href="github.com" target="_blank">
+                  <img src="./images/github.svg" alt="github-icon" />
+                </a>
+              </li>
+
+              <li>
+                <a href="angel.com" target="_blank">
+                  <img src="./images/angel.svg" alt="angel-icon" />
+                </a>
+              </li>
             </ul>
-        </div>
-    </section>
-    <section class="works">
-        <div class="cardworks1">
-            <div class="snapshot">
-                <img src="images/Snapshoot Portfolio.png" alt="snapshot portfolio">
-            </div>
-            <div class="leftblock">
-                <div class="prim-txt">
-                    <h2>Tonic</h2>
-                    <img src="images/Frame 2.png" alt="works">
+          </div>
+        </section>
+         </div>
+        <!-- Home-page section Ends here -->
+
+        <!-- Work-section Starts here -->
+
+        <section id="work-section">
+          <div class="grid-container">
+            <!-- portfolio - 1 -->
+            <div class="card">
```

Comment:
Very good job in this project 💪 you are almost done , **Only** one small issue to fix is the width of your cards  please try your pest to replicate the same design in the template otherwise you have done very well 👍

**figma : **

![temp1 margin left right](https://user-images.githubusercontent.com/59707859/220965740-6d5b4ab9-7bc0-4f0b-bc08-39e04990be3d.PNG)

**student : **

![student temp1 margin](https://user-images.githubusercontent.com/59707859/220965827-fbb1a43b-9cba-4fca-a9c7-f2df86fe1663.PNG)